Cubical/Reflection/StrictEquiv.agda

```diff
@@ -0,0 +1,53 @@
+{-# OPTIONS --cubical --no-import-sorts --safe #-}
+module Cubical.Reflection.StrictEquiv where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Function
+open import Cubical.Foundations.Equiv.Base
+open import Cubical.Foundations.Isomorphism
+
+open import Cubical.Data.List.Base
+open import Cubical.Data.Unit.Base
+
+import Agda.Builtin.Reflection as R
+open import Cubical.Reflection.Base
+
+strictEquivTerm : R.Term → R.Term → R.Term
+strictEquivTerm f g =
+  R.pat-lam
+    ( R.clause []
+        (R.proj (quote fst) v∷ [])
+        f
+    ∷ R.clause []
+        (R.proj (quote snd) v∷ R.proj (quote equiv-proof) v∷ [])
+        (R.def (quote strictContrFibers) (g v∷ []))
+    ∷ []
+    )
+    []
+
+strictEquivMacro : ∀ {ℓ ℓ'} {A : Type ℓ} {B : Type ℓ'}
+  → (A → B) → (B → A) → R.Term → R.TC Unit
+strictEquivMacro {A = A} {B} f g hole =
+  R.quoteTC (A ≃ B) >>= λ equivTy →
+  R.checkType hole equivTy >>
```

Comment:
Does it not work without this part? I thought the hole would already have the right type from the surrounding context.

Reply:
Without that part it still works but is a little less robust. For example in `Cubical.Data.Sigma.Properties` there's a definition 
```
ΣPathP≃PathPΣ = strictIsoToEquiv ΣPathPIsoPathPΣ
```
without a type signature, so there's no surrounding context. Without those lines it becomes necessary to add a type signature.

Reply:
That makes sense, nice to know.